pydantic/main.py

```diff
@@ -1425,6 +1425,7 @@ def create_model(
     if resolved_bases is not __base__:
         ns['__orig_bases__'] = __base__
     namespace.update(ns)
+    kwds['is_dynamic_model'] = True
```

Comment:
I don't think we need `is_dynamic_model` here, I think the `__module__: str = __name__,` parameter should work.

If for some reason that doesn't work, we'll need to use `f = sys._getframe(3)` here.